tests/test_tls.py

```diff
@@ -584,9 +584,12 @@ def test_handshake_with_ed25519_certificate(self):
         )
 
     def test_handshake_with_ed448_certificate(self):
```

Comment:
Why not use the same `skipIf("ed448" in SKIP_TESTS)` as above?

Reply:
Indeed it make things cleaner